docs/changes.rst

```diff
@@ -19,6 +19,7 @@ Pending
   <https://astral.sh/blog/the-ruff-formatter>`__.
 * Changed the default position of the toolbar from top to the upper top
   position.
+* Add a setting for refresh or not the toolbar if an ajax request occurs.
```

Comment:
```suggestion
* Added the setting, ``UPDATE_ON_FETCH`` to control whether the 
  toolbar automatically updates to the latest AJAX request or not.
  It defaults to ``False``.
```
Change the tense, added the setting name for folks and mentioned the default value.